SmartDeviceLink/SDLFunctionID.m

```diff
@@ -28,7 +28,7 @@ - (instancetype)init {
     if (!self) {
         return nil;
     }
-
+    
```

Comment:
Careful of adding extra whitespace. I'd recommend turning on Xcode->Preferences->Text Editing->Automatically trim trailing whitespace (and also Including whitespace-only lines)